app/src/main/java/org/imozerov/streetartview/ui/extensions/UiExtensions.kt

```diff
@@ -16,8 +16,8 @@ import org.imozerov.streetartview.R
  */
 val TAG = "UiExtensions"
 
-fun ImageView.loadImage(imagePath: String) {
-    if (imagePath.isNotBlank()) {
+fun ImageView.loadImage(imagePath: String?) {
+    if (imagePath?.isNotBlank() == true) {
```

Comment:
Just stupid question... :flushed: 
Does `imagePath?.isNotBlank()` return `null` when `imagePath` is `null`? That's why do we need to check `== true`?